src/main/kotlin/no/nav/syfo/db/PlanlagteVarslerDAO.kt

```diff
@@ -0,0 +1,78 @@
+package no.nav.syfo.db
+
+import no.nav.syfo.domain.PPlanlagtVarsel
+import no.nav.syfo.domain.PlanlagtVarsel
+import java.sql.Date
+import java.sql.Timestamp
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.*
+
+fun DatabaseInterface.storePlanlagtVarsel(planlagtVarsel: PlanlagtVarsel): UUID {
+
+    val insertStatement1 = """INSERT INTO PLANLAGTE_VARSLER (
+        uuid,
+        varseldato
+        type,
+        arbeidstaker_fnr,
+        orgnummer                        
+        opprettet,
+        sist_endret) VALUES (?, ?, ?, ?, ?, ?, ?)""".trimIndent()
+
+    val now = Timestamp.valueOf(LocalDateTime.now())
+    val varselUUID = UUID.randomUUID()
+
+    connection.use { connection ->
+        connection.prepareStatement(insertStatement1).use {
+            it.setObject(1, varselUUID)
+            it.setDate(2, Date.valueOf(planlagtVarsel.varselDato))
+            it.setString(3, planlagtVarsel.arbeidstakerFnr)
+            it.setString(4, planlagtVarsel.orgnummer)
+            it.setTimestamp(5, now)
+            it.setTimestamp(6, now)
+            it.executeUpdate()
+        }
+        connection.commit()
+    }
+    return varselUUID
+}
+
+fun DatabaseInterface.findDuplicateEntry(type: String, fnr: String, orgnr: String): PPlanlagtVarsel? {
+    val queryStatement = """SELECT *
+                            FROM PLANLAGTE_VARSLER
+                            WHERE type = ? and arbeidstaker_fnr = ? and orgnummer = ?
```

Comment:
Her er det visst en typo. Skal være `arbeidstakerFnr` i stedet for `arbeidstaker_fnr`